src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -232,11 +234,20 @@ library BeaconChainProofs {
             "BeaconChainProofs.verifyWithdrawal: historicalSummaryIndex is too large"
         );
 
-        require(
-            withdrawalProof.withdrawalProof.length ==
-                32 * (EXECUTION_PAYLOAD_HEADER_FIELD_TREE_HEIGHT + WITHDRAWALS_TREE_HEIGHT + 1),
-            "BeaconChainProofs.verifyWithdrawal: withdrawalProof has incorrect length"
-        );
+        if(block.timestamp >= denebForkTimestamp){
```

Comment:
The if statement with 2 identical requires is kinda messy. Can we separate this out into an if statement + a single require? Something like this:

```
uint256 withdrawalProofLen;
if (block.timestamp >= denebForkTimestamp) {
  withdrawalProofLen = /**deneb calc here*/;
} else {
  withdrawalProofLen = /**capella calc here*/;
}

require(withdrawalProof.withdrawalProof.length == withdrawalProofLen, "err here");
```